api/paths/paths.go

```diff
@@ -1,80 +1,138 @@
-package types
+package paths
 
 import (
 	"bytes"
 	"fmt"
 	"io"
+	"log"
```

Comment:
Hi @codenrhoden,

For this package I specifically wanted to make sure it was *only* using stdlib due to the fact it will likely be the very first package anything invokes. I wanted to make sure it did not have dependencies on anything else.